app/i18n/nl/admin.php

```diff
@@ -112,6 +112,13 @@
 		),
 		'title' => 'Uitbreidingen',
 		'user' => 'Gebruikersuitbreidingen',
+		'community' => 'Available community extensions for download', // @todo translate
```

Comment:
Gebruikersuitbreidingen beschikbaar voor download

PS You don't need to add the Dutch translations but I figured since you're on it. :-P